src/Simple.OData.Client.Core/Fluent/FluentCommand.cs

```diff
@@ -14,7 +14,7 @@ namespace Simple.OData.Client
     // For the same reason FluentClient is also declared as public
     // More: http://bloggingabout.net/blogs/vagif/archive/2013/08/05/we-need-better-interoperability-between-dynamic-and-statically-compiled-c.aspx
 
-    public partial class FluentCommand
+    public class FluentCommand
```

Comment:
Why was `partial` dropped?

Reply:
The implementation of FluentCommand contains only this single part so `partial` is not necessary here. This change is not related to my other changes so I can revert this.